contrib/deploy.sh

```diff
@@ -1,3 +1,18 @@
 #!/bin/bash
 
 export KUBECONFIG="$(kind get kubeconfig-path)"
+
+echo "Applying namespaces"
+kubectl apply -f ./namespaces.yml
+
+PASSWORD="something_random"
```

Comment:
Hmm? What about using something random and storing it in a file?

Reply:
I wanted to make the test easy to follow. Saving it a file seemed like a bit complex 🤔  it can be done later if there is any issue. I am also using the password to login in faas-cli in `run_function.sh`

Reply:
Please store it in a file and use it again?